packages/xod-client/src/core/styles/components/Helpbox.scss

```diff
@@ -0,0 +1,59 @@
+.Helpbox {
+  position: fixed;
+  z-index: 5;
+  top: 0;
+  left: $sidebar-width;
+  background: $sidebar-color-bg;
+  border: 1px solid $chrome-outlines;
+  border-radius: 5px;
+  box-shadow: 5px 5px 10px -5px rgba(0,0,0,.5);
+  max-height: 80%;
+
+  margin-top: -13px; // to point accurately on selected item
+
+  display: flex;
+  flex-direction: column;
+
+  transition: opacity 0.1s;
+
+  &--hidden {
+    opacity: 0;
+    pointer-events: none;
+  }
+
+  &-content {
+    padding: 12px;
+    overflow: auto; // TODO
```

Comment:
TODO what? Does this thing cause native scrollbars?

Reply:
Yep, forget it 😬